kanopi/starter-8
# Clean gitignore

## Description
> As a developer, I want a clean repo, so that I can develop faster, review cleaner code, and generally be a happier human being.

Two tiny tweaks:
- `.idea` had two entries, so removing the second.
- Drupal's `.gz` files seemed to be _un-ignored_ which seems... not good? I ran `composer install` and I got a bunch of those files to commit to the repo. That seems wrong, right?

## Acceptance Criteria


## Affected URL
None

## Related Tickets
None

## Steps to Validate
1. Open project in PHPStorm, observe no `.idea` files are shown by git.
1. Run `composer install`. Observe no new files are shown by git.


## Deploy Notes
NA

**New scripts**:

- `script` : script details

**New dependencies**:

- `dependency` : dependency details

**New dev dependencies**:

- `dependency` : dependency details
